courseraprogramming/commands/upload.py

```diff
@@ -347,6 +356,27 @@ def parser(subparsers):
              'also be associated with the grader.')
 
     parser_upload.add_argument(
+        '--grader-cpu',
+        type=int,
+        choices=[1, 2],
+        help='Amount of CPU your grader is allocated. You may choose from 1 '
+             'or 2 full CPU cores. The default number is 1.')
+
+    parser_upload.add_argument(
+        '--grader-memory-limit',
+        type=int,
+        choices=[1024, 2048],
+        help='Amount of memory your grader is allocated. You may choose from '
+             '1024 MB or 2048 MB. The default amount is 1024 MB.')
+
+    parser_upload.add_argument(
+        '--grading-timeout',
+        type=lambda v: utils.check_int_range(v, TIMEOUT_LOWER, TIMEOUT_UPPER),
+        help='Amount of time allowed before your grader times out, in '
```

Comment:
I think that's a little too verbose for command line help.  I'll add it to the readme though.